pkg/component/component.go

```diff
@@ -99,7 +100,8 @@ func GetComponentPorts(client *occlient.Client, componentName string, applicatio
 
 // CreateFromPath create new component with source or binary from the given local path
 // sourceType indicates the source type of the component and can be either local or binary
-func CreateFromPath(client *occlient.Client, name string, componentImageType string, path string, applicationName string, sourceType string, inputPorts []string) error {
+// envs is the array containing the string env values
+func CreateFromPath(client *occlient.Client, name string, componentImageType string, path string, applicationName string, sourceType string, inputPorts []string, envVars []string) error {
```

Comment:
These are way too many functions going into CreateFromPath (max 6 for Go is acceptable...)

I've refactored this in #622 but either way, one of us will have to rebase. Just doing a note for this in the future.

Reply:
Ok if your PR gets in before this, then I will rebase